### ISSUE TITLE
keeweb desktop executable blocked by Symantec Endpoint Protection

### ISSUE THREAD

**Issue by sach2** [keeweb/keeweb#118]
When i tried to execute keeweb desktop installer, it was blocked and immediately deleted by Symantec Endpoint protection antivirus.

**Comment by antelle** [comment]
Does it provide any info, _why_ it's blocking the app?

**Comment by sach2** [comment]
Following information is logged.
Risk type - Insight Network Threat
Risk - WS.Reputation.1.
Following link was shown for more information - http://securityresponse.symantec.com/security_response/writeup.jsp?docid=2010-051308-1854-99&vid=4294919973

**Comment by antelle** [comment]
So, it's Symantec heuristic:
> it does not represent a specific class of threat like adware or spyware, but instead applies to all threat categories. 

I don't know how do deal with it.

**Comment by cdsimpkins** [comment]
Sounds like the problem in this case is lack of a large user base.  At least that's how I interpret it. 

> for files that have a low reputation score based on analyzing data from Symantec’s community of users

**Comment by antelle** [comment]
I've had a talk about this with folks.
They say, the way it works is:
- digital signature might matter 
- as it's electron, and not the native app, anti-viruses are more aggressive to it
- maturity matters as well (currently this app is a new one)
- we may contact them and provide some details but the signature is mandatory here

From Symantec description:
The system considers many aspects of a file, including
- file age (not so ok: the file is new)
- file download source (ok, I guess, as the source has https)
- digital signature (fail; we don't have it currently)
- file prevalence (not ok)

So, I think the first thing to start with is signing the software (which is not easy because I'm searching for extremely cheap options, below $10 per year). As the time passes by, anti-viruses become more loyal. And we can speed up this process by contacting them.

**Comment by antelle** [comment]
Closing as a duplicate of #196: once we get signed, it will be possible to move further if necessary but this will not happen unless we get stable investments of $400+/yr. For now, the only solution is to add an exception.

**Comment by Neo13Pr** [comment]
I got same "Symantec Endpoint Protection" issue while installing the app, and for that i need to sign my code, so could anyone please tell me how can we use this "certificateSubjectName" variable in our electron code and where? and how to sign electron app, I am using windows platform to create a build.

Thanks.
